db/seeds.rb

```diff
@@ -7,15 +7,15 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 
-adelle = User.create(user_name: "Adelle", location: "Maine", email_address: "adelle@gmail.com")
+adelle = User.create(user_name: "Adelle", location: "Maine", email_address: "adelle@example.com")
```

Comment:
You've already got great seeds but keep in mind you can use factorybot for seed data as well!